gui/services/forms.py

```diff
@@ -132,6 +132,11 @@ def save(self, *args, **kwargs):
             else:
                 started = _notifier.stop("domaincontroller")
 
+        elif obj.srv_service == 'afp' and _notifier._started_domaincontroller():
+            obj.srv_enable = True
+            obj.save()
+            started = True
+
```

Comment:
I just tried implementing AFP service as CIFS, however testing the code without these lines also work, So they aren't required. I'll make the required changes.